rfps/epassport-zk-validation.md

```diff
@@ -0,0 +1,51 @@
+# e-Passport ZK Validation
```

Comment:
Can someone be added as a proposer to this one? Looks like they had some domain-specific knowledge which may prove useful during the application review.

Reply:
@burdges @FlorianFranzen apparently

Reply:
https://github.com/arkworks-rs/curves/issues/82 
https://github.com/LogvinovLeon/estid-sig/issues/5